feature-toggle/src/app/feature.service.ts

```diff
@@ -0,0 +1,45 @@
+import { Injectable } from '@angular/core';
+import { environment } from '../environments/environment';
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+import { Observable, Subject, tap } from 'rxjs';
+import { LoginAccept, LoginForm, SignUpAccept, SignUpForm } from './interface/feature.interface';
+
+
+@Injectable({
+  providedIn: 'root'
+})
+export class FeatureService {
+  private baseUrl = environment.apiUrl;
+  public userId: number = 0;
+
+  constructor(private http: HttpClient) {
+    // Retrieve userId from sessionStorage on service initialization
+    // const storedUserId = sessionStorage.getItem('userId');
+    const storedUserId = 0; 
+    if (storedUserId) {
+      this.userId = parseInt(storedUserId, 10); // Parse the stored string value to a number
+    }
+  }
+
+
+
+  
+  login(data: LoginAccept): Observable<any> {
```

Comment:
dont use any

Reply:
> dont use any
login and addUser function will be fixed after merging